moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -284,6 +284,15 @@ bool Servo::validateParams(const servo::Params& servo_params) const
     params_valid = false;
   }
 
+  if (servo_params.max_expected_latency / 3 < servo_params.publish_period)
```

Comment:
This `3` in here is a magic number, but I'm assuming this is related to you wanting 3 points in your trajectory, right? Is this the same as `MIN_POINTS_FOR_TRAJ_MSG` you defined in `common.cpp`?

If that's not the case, you can probably leave in the literal here (especially since it's also in the error message), but definitely add a comment explaining.

Reply:
I think it's correct to relate the 3 here to `MIN_POINTS_FOR_TRAJ_MSG`. Do you think I should move `MIN_POINTS_FOR_TRAJ_MSG` to `moveit_servo/utils/common.hpp` so it can be accessed?

Reply:
I think so!